src/sentry/api/endpoints/debug_files.py

```diff
@@ -428,14 +429,15 @@ def expose_release(release, count):
             }
 
         def serialize_results(results):
-            file_counts = (
-                Release.objects.filter(id__in=[r["id"] for r in results])
-                .annotate(count=Count("releasefile"))
-                .values("count", "id")
+            # Using raw SQL here because Django 1.11 does not support filters
+            # in ``Count`` annotation yet
+            file_counts = ReleaseFile.objects.raw(
+                "SELECT release_id AS id, count(*) FROM sentry_releasefile WHERE release_id = ANY(%s) AND name != %s GROUP BY release_id",
+                params=([r["id"] for r in results], RELEASE_ARCHIVE_FILENAME),
```

Comment:
If I understand correctly here, all you care about is filtering out `ReleaseFile` rows with a specific name.

I think `ReleaseFile.objects.filter(release_id__in=[1,2,3]).exclude(name='bad').values("release_id").annotate(count=Count("id"))` will do what you want. 

Produces:
```
SELECT "sentry_releasefile"."release_id", COUNT("sentry_releasefile"."id") AS "count" 
FROM "sentry_releasefile" 
WHERE ("sentry_releasefile"."release_id" IN (1, 2, 3) AND NOT ("sentry_releasefile"."name" = bad)) 
GROUP BY "sentry_releasefile"."release_id"
```

Reply:
@wedamija Sorry, I saw your comment just a couple of seconds too late. I think it makes sense to close this PR as this query is solved differently in the next one anyway. Nevertheless thank you for the review!